crates/subspace-farmer-components/src/plotting.rs

```diff
@@ -341,6 +347,9 @@ where
                 });
         });
 
+    sector_output.resize(sector_size, 0);
```

Comment:
Would it be more efficient if we only resized in case the vector is empty. 

From what I see in the `resize` implementation, rust lib will still call truncate even though size is same. https://doc.rust-lang.org/src/alloc/vec/mod.rs.html#2355

Reply:
I don't think it is worth the complexity and compiler will likely optimize those things anyway. This is not a hot part of the code to add extra if IMHO.